src/lib/registration/widgets/registration_code.rb

```diff
@@ -61,20 +61,19 @@ def store
       def register
         if skip?
           log.info("Empty value, skipping registration")
-          return true
+          return false
```

Comment:
As you commented as we are not taking take care of the output does not affect, but in case of already registered then register will not be called or just return false.